app/assets/javascripts/charts.js

```diff
@@ -0,0 +1,21 @@
+$(function() {
+  $('.row a').on('click',  function(event)  {
+    event.preventDefault(); // Prevent link from following its href
+    var previous_poll_id = $('#previous-poll').data('poll_id');
+    var next_poll_id = $('#next-poll').data('poll_id');
+    var get_url;
+    if(this.id=="previous_graph") {
+      get_url = "api/polls/" + previous_poll_id;
+    }
+    else {
+      get_url = "api/polls/" + next_poll_id;
+    }
+
+    $.get(get_url, function(data){
+      data: $(event.target).serialize();
+            success: new Chartkick.PieChart($('.chart')[0], data.chart_data);
```

Comment:
Same problem down here. This entire containing anonymous function (starting on like 14) is actually the success handler. This line here is just being read as another label. The code on the right side is indeed being run, which is why the chart is still changing, but the `success:` can go, and we can fix the indentation.